archway-1/gentx/R1M.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"R1M","identity":"BC69E82C942524D3","website":"https://r1m.team/","security_contact":"r1m.nodes@gmail.com","details":"Validator service, 24/7 availability"},"commission":{"rate":"0.010000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.050000000000000000"},"min_self_delegation":"1","delegator_address":"archway1m52wf2046uvjer83mu5h2cny8xgqt5r7ejkdju","validator_address":"archwayvaloper1m52wf2046uvjer83mu5h2cny8xgqt5r7ekl0l4","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"bq5W8T3Ett3/P+Ks8iEg8mhyOxMfeb0DXlulVOPe9ew="},"value":{"denom":"ustate","amount":"9000000000"}}],"memo":"208131a7627ea2f6ff55a496360b9bd274126dd0@95.217.4.86:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A09hmIouz2jcbuqTiXWLIza3urdhXqPNaz0mm2LcZwVb"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["Yo5062yMpEhZ2t0HtrKaDZfp2dJnwZJalOYuMHEfs8EJ7tdTaQB3EvTKHoNExvpKVhykwwNpcVt5ndgvNIxjhw=="]}
```

Comment:
Validation fails with message
```
Gentx fee is empty
```
@sorkand1 ensure you set fee according to https://docs.archway.io/validators/becoming-a-validator/running-a-validator-node#create-your-validator